snuba/util.py

```diff
@@ -45,6 +46,14 @@ def to_list(value):
     return value if isinstance(value, list) else [value]
 
 
+def qualified_column(column_name: str, alias: str="") -> str:
+    """
+    Returns a column in the form "table.column" if the table is not
+    empty. If the table is empty it returns the column itself.
+    """
+    return column_name if not alias else f"{alias}.{column_name}"
```

Comment:
Do you think this will evolve into something where this is a data type in the `Query` class itself, rather than something that is applied functionally in various places? Or some other long-term plan?

Reply:
yes definitively. 
The reason why we need this is that expressions in the current Snuba are passed around as string instead of being abstract data structures. Once we will move there, logic like this will be in the last stage of formatting the query.

Reply:
This is also kind of what I was getting at with https://github.com/getsentry/snuba/pull/474/files/6a60ec84a117c6ecd640d7db3e07a17b00ed0507#r328877789.